File: src/main/java/factorio/debugger/DAP/messages/requests/DAPConfigurationDoneRequest.java
package factorio.debugger.DAP.messages.requests;

import com.fasterxml.jackson.annotation.JsonTypeName;
import factorio.debugger.DAP.messages.DAPAdditionalProperties;
import factorio.debugger.DAP.messages.DAPRequest;

@JsonTypeName("configurationDone")
public class DAPConfigurationDoneRequest extends DAPRequest<DAPConfigurationDoneRequest.ConfigurationDoneArguments> {
    ConfigurationDoneArguments arguments;

    public DAPConfigurationDoneRequest() {
        arguments = new ConfigurationDoneArguments();
    }

    public static class ConfigurationDoneArguments extends DAPAdditionalProperties {
    }
}

File: src/main/kotlin/factorio/debugger/DAP/messages/requests/DAPConfigurationDoneRequest.kt
package factorio.debugger.DAP.messages.requests

import com.fasterxml.jackson.annotation.JsonTypeName
import factorio.debugger.DAP.messages.DAPAdditionalProperties
import factorio.debugger.DAP.messages.requests.DAPConfigurationDoneRequest.ConfigurationDoneArguments

@JsonTypeName("configurationDone")
class DAPConfigurationDoneRequest : DAPRequest<ConfigurationDoneArguments>(ConfigurationDoneArguments()) {

    class ConfigurationDoneArguments : DAPAdditionalProperties()
}
